assets/js/components/dashboard-sharing/DashboardSharingSettingsButton.js

```diff
@@ -95,7 +108,7 @@ export default function DashboardSharingSettingsButton() {
 						<Button
 							aria-label={ __( 'Back', 'google-site-kit' ) }
 							className="googlesitekit-dialog__back"
-							onClick={ closeDialog }
+							onClick={ () => closeDialog( true ) }
```

Comment:
Same here. We don't have to have the parameter.

Reply:
Sorted in light of latest review. Thank you @hussain-t!